packages/backend/src/modules/user/user.repository.ts

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-underscore-dangle */
+import { CreateUserDto, PatchUserDto } from 'shared'
+import { User } from './user'
+import UserModel from './user.model'
+import UserMapper from './user.mapper'
+
+class UserRepository {
+  private readonly _model
+
+  constructor() {
+    this._model = UserModel
+  }
+
+  async create(dto: CreateUserDto): Promise<User> {
+    const newUser = await this._model.create(dto)
+    return UserMapper.mapToDomain(newUser)
```

Comment:
Mappers should be used in controllers. Inside of services we could still (for example) need to use some methods of mongoose

Reply:
Thanks, removed all mappers from here and added `null` handling